commands/FBVisualizationCommands.py

```diff
@@ -57,6 +57,50 @@ def _showImage(commandForImage):
     imgFile.close()
     os.system('open ' + imagePath)
 
+def _colorIsCGColorRef(color):
+  color = '(CGColorRef)(' + color + ')'
+
+  frame = lldb.debugger.GetSelectedTarget().GetProcess().GetSelectedThread().GetSelectedFrame()
+  result = frame.EvaluateExpression('(unsigned long)CFGetTypeID({color}) == (unsigned long)CGColorGetTypeID()'.format(color=color))
+
+  if result.GetError() is not None and str(result.GetError()) != 'success':
+    print "got error: {}".format(result)
+    return 0
+  else:
+    isCFColor = result.GetValueAsUnsigned() != 0
+    if isCFColor:
+      return 1
+    else:
+      return 0
```

Comment:
Cone to think of it, the `!= 0` is already returning a boolean. We can just return that.